src/plugins/guided_onboarding/public/services/api.service.ts

```diff
@@ -169,41 +169,32 @@ export class ApiService implements GuidedOnboardingApi {
   }
 
   /**
-   * Activates a guide by guideId.
-   * This is useful for the onboarding landing page, when a user selects a guide to start or continue.
+   * Activates a guide's default state by guideId.
+   * This is useful for defining an initial state for a guide.
    * @param {GuideId} guideId the id of the guide (one of search, kubernetes, siem)
    * @param {GuideState} guide (optional) the selected guide state, if it exists (i.e., if a user is continuing a guide)
    * @return {Promise} a promise with the updated plugin state
    */
-  public async activateGuide(
+  public async activateGuideDefaultState(
```

Comment:
Moving forward, I think we have to be careful when we rename public methods in the API service. I think this is only used on the landing page, so it's probably OK, but soon there are going to be other teams consuming the API service and I'd like us to be thoughtful of that.